installer/win/WixSharpInstaller/CloseDialog.cs

```diff
@@ -32,22 +29,21 @@ void closeButton_Click(object sender, System.EventArgs e)
         {
             // The "Close Symphony" button is just to get users consent to close the app.
             // Actually closing the app will be done later in the flow.
-            Shell.GoTo(InstallDirIndex);
-        }
-
-        void back_Click(object sender, System.EventArgs e)
-        {
-            Shell.GoTo(WelcomeDlgIndex);
+            Shell.GoNext();
         }
 
         void next_Click(object sender, System.EventArgs e)
         {
-            Shell.GoTo(InstallDirIndex);
+            Shell.GoNext();
         }
 
         void cancel_Click(object sender, System.EventArgs e)
         {
-            Shell.Cancel();
+            if( System.Windows.Forms.MessageBox.Show("Are you sure you want to cancel Symphony installation?",
```

Comment:
we should perhaps get this string from a strings file to support internationalisation in the future.

Reply:
I agree that it would be better, but we have not planned to do localization until phase 2 of installer, and figuring out and setting up to get this string from localization files would mean doing a big part of that work now, so I think that it is better to leave it for now. But I think I will add a "TODO: localization" comment to the relevant places in the code, so we can find it when we do localization